README.md

```diff
@@ -10,13 +10,22 @@ As a user, you will need to create a new shopping list or join an existing list.
 
 ## Video demo of the app:
 
-[![](https://cdn.zappy.app/33815167c45d74c3ae5af232de633add.png)](https://www.youtube.com/watch?v=mwj74mE9s64&t)
+![](https://github.com/the-collab-lab/tcl-19-smart-shopping-list/blob/read-me/public/smart-shopping-app.gif)
 
 
 ## Live project
 
 The live project can be viewed [here](https://tcl-19-smart-shopping-list-2.web.app/).
 
+## Setting up the project
+
+- This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app)
+- [Google Firebase](https://firebase.google.com/) is used to store the data
+- To make integration with Firebase even more straightforward [react-firebase-hooks](https://firebaseopensource.com/projects/csfrequency/react-firebase-hooks/) were installed
+- To navigate between the different views [react-router-dom](https://reactrouter.com/web/guides/quick-start) was installed
+- For styling [TailwindCSS](https://tailwindcss.com/docs/guides/create-react-app) was chosen
+
+
```

Comment:
Thanks @YuriDevAT  for the PR. It looks like there is mismatch between the heading and content. All the bullet points seem to talk about dependencies except the first one. Wouldn't it be better to call it **Dependencies**  instead of **Setting up the project**?

Reply:
True. Should I just go with "Dependencies" or with something like "Dependencies installed (or "used")"?

Reply:
Also, what do you think about adding "Contributors" at the end of the Readme? I know we have it in an extra file, with mentors etc. included, but we could put our names in the readme as well (I see this sometimes in other readmes)

Reply:
> True. Should I just go with "Dependencies" or with something like "Dependencies installed (or "used")"?

I think **Dependencies Used** sounds fine



> Also, what do you think about adding "Contributors" at the end of the Readme? I know we have it in an extra file, with mentors etc. included, but we could put our names in the readme as well (I see this sometimes in other readmes)

That would be awesome. It would be great if you could include contributor's avatar as well.

I think it would also be great to include the fancy app thumbnail you created the other time in the README. 

Could you point the live project link to the working version I deployed https://smart-shopping-list.netlify.app/ until we figure out how to use environment variables in firebase hosting? I will try to set up github actions to sync my forked repository with this repository periodically.

Thanks for the good job you are doing

Reply:
I was wondering why the netlify link still works :D 
Sure, with avatars would be great :)
About the thumbnail, just the thumbnail OR both, gif and thumbnail? (This seemed a bit too much, idk)
For now, I also have to figure out something about the failing deploy 
<img width="863" alt="Screenshot 2021-05-17 at 09 47 29" src="https://user-images.githubusercontent.com/54622834/118508604-f4c8ae00-b6f4-11eb-9231-34e1a04b74f2.png">

Reply:
You have quite a number of commits. Probably I missed the gif commit. Will the gif be at the top?

About the failing deploy, don't worry. I also experienced it too. There must be something wrong with the CI/CD config file. When the PR is merged the build succeeds. The error message suggests we need to run `npm ci` before running the build job.
![image](https://user-images.githubusercontent.com/52580190/118510185-63c7f600-b739-11eb-9432-e937046872ae.png)

Reply:
Ok good to know, I already was afraid of merging :D I inserted the thumbnail again. We could give it under the "overview" heading e.g.